### PR TITLE
build: enable ckBTC ledger and index for mainnet

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Enable ckBTC ledger and index on mainnet and propose it in next version.

# Changes

- enable ckBTC on **mainnet**
- enable ckBTC per default
- integrate ckBTC canister IDs configuration and fallback to mainnet canister IDs
- update order of fields in script
- add a new utils to test if a string is not null, not undefined and not empty

### CODE REVIEW

File: config.sh [hunk 1]
```diff
@@ -139,8 +139,8 @@ VITE_FEATURE_FLAGS=$featureFlags
 VITE_HOST=$host
 VITE_IDENTITY_SERVICE_URL=$identityServiceUrl
 VITE_AGGREGATOR_CANISTER_URL=${aggregatorCanisterUrl:-}
-VITE_CKBTC_INDEX_CANISTER_ID=${ckbtcIndexCanisterId:-}
-VITE_CKBTC_LEDGER_CANISTER_ID=${ckbtcLedgerCanisterId:-}" | tee "$ENV_FILE"
+VITE_CKBTC_LEDGER_CANISTER_ID=${ckbtcLedgerCanisterId:-}
```
Comment (lmuntaner) [inline config.sh:142]: Why did yo change the order?

**Review by lmuntaner — APPROVED** [review]
LGTM!

File: config.sh (hunk 1 above)
Comment (peterpeterparker) [reply config.sh:142]: For readability consistency reason. Everywhere we use ledger then index except here where it was index then ledger

Comment (peterpeterparker) [reply config.sh:142]: Thanks for the review!
